### PR TITLE
GUI client dashboard and create client

### PR DESCRIPTION
**Pull request by sgtho**
updated the new GUI work.
 
What I did:
1. Made the client window connect with clients.jsonl, it shows the clients right now
2. Built the Ceate Client new window, and it successfully save the new record into the clients.jsonl
3. Close the new window without saving any record when you press the "Close" button

Something might be taken care for Tzelepidou, Maria Hui, Yau Kam :
When the new client has been created the id and type goes to the last order. It seems ok when running the main.py

### CODE REVIEW

File: src/gui/__init__.py [hunk 1]
```diff
@@ -14,45 +14,166 @@ def start(self):
...
+                "Name": name_entry.get(),
+                "Phone_Number": phone_entry.get(),
+                "Address_Line_1": line1_entry.get(),
+                "Address_Line_2": line2_entry.get(),
+                "Address_Line_3": line3_entry.get(),
+                "City": city_entry.get(),
+                "State": state_entry.get(),
+                "Zip_Code": zip_entry.get(),
+                "Country": country_entry.get(),
+                "Type": "Customer"  # Assuming all clients are "Customer"
```
Comment (mariatzele) [inline src/gui/__init__.py]: I made this mistake before actually. I think this should be "Client" not customer sorry

File: src/gui/__init__.py [hunk 2]
```diff
@@ -14,45 +14,166 @@ def start(self):
...
+        header_frame = ctk.CTkFrame(master=table_frame)
+        header_frame.pack(fill="x", pady=10)
+
+        headers = ["Client ID", "Type", "Name", "Address Line 1", "City", "State", "Zip Code", "Country", "Phone No."]
+        for header in headers:
+            label = ctk.CTkLabel(master=header_frame, text=header, width=10, font=("Arial", 10, "bold"))
+            label.pack(side="left", padx=10)
+
+        # Client Data
+        data_frame = ctk.CTkFrame(master=table_frame)
```
Comment (mariatzele) [inline src/gui/__init__.py]: There's a bug now where if we add a client it won't be shown in the table unless the app is closed and re-opened again. To fix this we can store this `data_frame` in `self` and then have an update method that removes the table's children and re-adds them by `calling client_repo.list()`

**Review by sgtho — COMMENTED** [review]
Change Type Customer to Client
Add view button to open the record.
Add edit and update function
Add delete function
Add confirmation pop up window

**Review by mariatzele — APPROVED** [review]

**Comment by mariatzele** [comment]
Great work, there's some minor bugs like the window doesn't close when we click "confirm" on update and the table doesn't refresh after a customer update but we can fix those in subsequent pull requests.

**Pull request merged**